subprojects/docs/src/docs/userguide/dependencyManagementWorkingWithDependencies.adoc

```diff
@@ -0,0 +1,87 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[working_with_dependencies]]
```

Comment:
:x: Note that the filename `dependencyManagementWorkingWithDependencies.adoc` needs to match the generated URL (currently `working_with_dependencies.html`) otherwise this will break the "Edit this page" widget.

Reply:
done